scala-mode2.el

```diff
@@ -169,5 +170,11 @@ When started, runs `scala-mode-hook'.
                '("\\.\\(scala\\|sbt\\)\\'" . scala-mode))
   (modify-coding-system-alist 'file "\\.\\(scala\\|sbt\\)\\'" 'utf-8))
 
+(defvar scala--prettify-symbols-alist scala-mode-pretty-all-alist)
```

Comment:
I saw this [here](http://emacsredux.com/blog/2014/08/25/a-peek-at-emacs-24-dot-4-prettify-symbols-mode/)